### PR TITLE
Added basic info to the admin guide

### PR DESCRIPTION
**Pull request by maximAtanasov**
and removed outdated references from the porperties template

### CODE REVIEW

File: coderadar-test/src/main/asciidoc/admin.adoc [hunk 1]
```diff
@@ -12,89 +12,85 @@ v{version}, {date}
...
 
-=== Deploying coderadar on an application server
+Adjusting this value will change how often projects are checked for new commits/branches.
 
-WARNING: This documentation is work in progress. If you want to contribute, have a look at this task in the
- issue tracker: https://github.com/reflectoring/coderadar/issues/148.
+==== Configuring the working directory
 
-== Clustering
+This is the directory project will be cloned to. It's path can be adjusted by changing the following property:
```
Comment (KilianKrause) [inline coderadar-test/src/main/asciidoc/admin.adoc]: This is the directory **projects** will be cloned to.

File: coderadar-test/src/main/asciidoc/admin.adoc [hunk 2]
```diff
@@ -12,89 +12,85 @@ v{version}, {date}
...
-  the jobs waiting in queue.
+The log file's path can be adjusted with the following property:
 
-| coderadar.jobs.failed.*
-| Count and rate of failed jobs.
+`logging.file=coderadar.log`
 
-| coderadar.jobs.finished.*
-| Count and throughput of successfully finished jobs.
+==== Configuration the database connection
```
Comment (KilianKrause) [inline coderadar-test/src/main/asciidoc/admin.adoc]: Either **Configuring the database connection** or **Configuration of the database connection**.

File: coderadar-test/src/main/asciidoc/admin.adoc [hunk 3]
```diff
@@ -12,89 +12,85 @@ v{version}, {date}
...
+The easiest way to quickly configure a local Neo4j instance is by using Neo4j Desktop.
+
+Once started, the Coderadar REST-API will be available on port 8080.
+For more information on it, have a look at the REST-API documentation:
+
+https://reflectoring.github.io/coderadar/current/docs/restapi.html
+
+=== Starting the Coderadar UI
+
+To start the Coderadar UI, you must first build it. This requires having NPM installed, going to the `coderadar-ui` directory and
```
Comment (KilianKrause) [inline coderadar-test/src/main/asciidoc/admin.adoc]: Suggestion: instead of **you must first build it** write **you have to build it first**.

Comment (maximAtanasov) [reply coderadar-test/src/main/asciidoc/admin.adoc]: Done

File: coderadar-test/src/main/asciidoc/admin.adoc (hunk 2 above)
Comment (maximAtanasov) [reply coderadar-test/src/main/asciidoc/admin.adoc]: Done

File: coderadar-test/src/main/asciidoc/admin.adoc (hunk 1 above)
Comment (maximAtanasov) [reply coderadar-test/src/main/asciidoc/admin.adoc]: Done

**Pull request merged**